### DISCUSSION TITLE
How to fully uninstall from Windows?

### DISCUSSION THREAD

**Discussion by cantcodetho** [w4ffl35/krita_stable_diffusion#62] [General]
Hi there.
Did not succeed in running this and now I would like to 100% remove every single piece of file from whatever install.bat installed. There is no uninstall section for Windows anywhere to my knowledge. Can you help with that?

**Comment by w4ffl35** [comment]
Hi @cantcodetho - this question slipped past me, sorry about the late reply.

1. Pres `Windows Key + E` to open a file explorer window
2. Paste `%APPDATA%\krita\pykrita\` into the path field
![image](https://user-images.githubusercontent.com/25737761/202173051-e278f6e0-e59c-427b-b675-d96f594f8de5.png)
3. Delete `krita_stable_diffusion` and `krita_stable_diffusion.desktop`
4. Paste `%USERPROFILE%\stablediffusion` into the path field
5. If this folder contains files or folders you wish to save, make a backup of those files
6. Delete `%USERPROFILE%\stablediffusion` (or cherry-pick that which you wish to keep)

**Comment by cantcodetho** [comment]
I forgot myself. Anyway thanks for the info.
